backend/app/controllers/api/v1/project_developers_controller.rb

```diff
@@ -31,6 +45,17 @@ def fetch_project_developer
         account = Account.friendly.find(params[:id])
         ProjectDeveloper.find_by!(account_id: account.id)
       end
+
+      def account_params
+        params.fetch(:project_developer_params, params)
```

Comment:
this is one of the rswag bugs which is fixed by this not super nice hack (https://github.com/rswag/rswag/issues/348). Unfortunately when using `formData`, rswag automatically adds name of the variable to param which in this case is `project_developer_params `